### ISSUE TITLE
RosterFilter not working on 1.12.1 Client

### ISSUE THREAD

**Issue by Mingrien** [coryo/RosterFilter#1]
This addon was working like charmed for a long time and then it just stopped working. I didn't use it for several months because it wasn't working. But i didn't find any other suitable addon for my use so i'd really want this working. I get a huge list of errors when i log in and can't just use the addon at all.

Screenshot of errors: https://ibb.co/dm2wsH

Please fix this, would be my life savior!

**Comment by coryo** [comment]
it seems like something isn't loading properly or there's a conflict with another addon. I've been using the current version without any problems.

Maybe try disabling some other things, or reinstalling it to make sure all the files are there.

**Comment by Mingrien** [comment]
Yeah i reinstalled it after a while just to see if it works now, i also installed AUX again. But yeah, i tried to disable ALL addon except Aux and Roster Filter and i get the same errors. Does the addon need some other addons to work? Or is there something you need to do for the folder inside Addons? Except r emoving the "-master" from there.

**Comment by coryo** [comment]
I'd guess if you disable aux or use an older version it would probably work.  I haven't updated aux in a long time and there was probably some changes that are causing a conflict between the two addons.

**Comment by Mingrien** [comment]
Ohh lol, it was the Aux. Well yeah, ty anyways! Would be nice to have it working at the same time tho..
